lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -199,4 +254,15 @@ export class ScopeSet {
         }
         return "";
     }
+
+    /**
+     * Prints scopes into a space-delimited string
+     * @param scopes
+     */
+    static finalScopes(scopes: Array<string>): string {
```

Comment:
printScopes() already does this, not sure why we need another function.

Reply:
There was some issue using it initially. I would like to build more generic interface for the class ScopeSet.ts to fit node, whatever is added is temporary mitigation.

Reply:
#1390 to address this